docs/INSTALL_OMNIA.md

```diff
@@ -1,47 +1,121 @@
 # Install Omnia using CLI
 
-The following sections provide details on installing Omnia using CLI. If you want to install the Omnia appliance and manage workloads using the Omnia appliance, see [Install the Omnia appliance](INSTALL_OMNIA_APPLIANCE.md) and [Monitor Kubernetes and Slurm](MONITOR_CLUSTERS.md) for more information.
+The following sections provide details on installing Omnia using CLI.  
+
+To install the Omnia control plane and manage workloads on your cluster using the Omnia control plane, see [Install the Omnia Control Plane](INSTALL_OMNIA_CONTROL_PLANE.md) and [Monitor Kubernetes and Slurm](MONITOR_CLUSTERS.md) for more information.
 
 ## Prerequisites
-* Ensure that all the prerequisites listed in the [Preparation to install Omnia](PREINSTALL_OMNIA.md) are met before installing Omnia.
-* If there are errors when any of the following Ansible playbook commands are run, re-run the commands again. 
+* The login, manager, and compute nodes must be running CentOS 7.9 2009 OS.
```

Comment:
Is this still true about CentOS 7.9 2009?

Reply:
Yes, this is true.